packages/protocol/src/action/request/Invoke.ts

```diff
@@ -10,19 +10,46 @@ import { MalformedRequestError } from "./MalformedRequestError.js";
 import { Specifier } from "./Specifier.js";
 
 export interface Invoke extends InvokeRequest {
+    /** Timeout only relevant for Client Interactions */
     timeout?: number;
 }
 
 /**
  * Request invocation of one or more commands.
  */
-export function Invoke(definition: Invoke.Definition): Invoke {
+export function Invoke(definition: Invoke.Definition): Invoke;
+
+/**
+ * Request invocation multiple commands with defined options
+ */
+export function Invoke(options: Invoke.Definition, ...data: CommandData[]): Invoke;
```

Comment:
On first read it wasn't clear to me if this was a list of commands or a list of inputs.  I know the type name comes from the spec but could we maybe make the argument name something more meaningful?  E.g. "...commands: CommandData[]".